requirements.txt

```diff
@@ -6,7 +6,7 @@ joblib>=0.14.0
 pytest>=2.8.5
 pytest-cov>=2.2.1
 pytest-flake8>=0.5
-#tensorflow>=2.0.0
+tensorflow>=2.0.0
 opencv-python>=4.1.0
 h5py>=2.9.0
-git+https://github.com/BioroboticsLab/bb_binary.git@master#egg=bb_binary
+git+https://github.com/BioroboticsLab/bb_binary
```

Comment:
Is this necessary? Without the "egg" parameter, PIP used to not be able to know which package name belonged to that URL. Might have changed though